app/metal-controller-manager/api/v1alpha1/server_types.go

```diff
@@ -100,12 +101,16 @@ type ServerSpec struct {
 	Accepted          bool                    `json:"accepted"`
 }
 
+// ConditionPowerCycle is used to control the powercycle flow.
```

Comment:
Name is PowerCycle (capital C) and then function on event log is Powercycle (lowercase c). We should be consistent. PowerCycle has my vote, but not a blocker for me.

Reply:
sure thing, let me update, I wasn't sure what looks better